modules/installation/proc-install-rhdh-helm.adoc

```diff
@@ -21,10 +21,19 @@ The {product} Helm chart is available in the Helm catalog in Red Hat OpenShift D
 
 . Switch to *Developer* mode on your Red Hat OpenShift web console.
 . Click *+Add*.
+. Click **create a Project**
+. Choose a name for your project and click **Create**
 . From the *Developer Catalog* panel, click *Helm Chart*.
 . Search for _Developer Hub_ in the search bar and select the *{product}* card.
 . Click *Create*.
-. Copy the OpenShift router host from *Root Schema* -> *global* -> *clusterRouterBase*, and adjust the other values if needed.
+. Note the OpenShift router host from *Root Schema* -> *global* -> *clusterRouterBase*, which will be needed in the next section, and adjust the other values if needed.
++
+
+[NOTE]
+====
+If you did not configure a custom domain for your cluster, you will need to replace the *clusterRouterBase* with the base domain for your cluster, something like `apps.<clusterName>.com`.
```

Comment:
Consider revising to _...with the base domain for your cluster. For example: `apps.<clusterName>.com`._

Reply:
The files in the modules folder are outdated and not active. While we're not using them now, we might utilize the modules folder in the future.